__tests__/app.test.js

```diff
@@ -365,6 +365,26 @@ describe("/api/comments/:comment_id", () => {
   });
 });
 
+describe("/api/users", () => {
+  test("GET 200: Responds with array of all user objects.", () => {
+    const exampleUser = {
+      username: expect.any(String),
+      name: expect.any(String),
+      avatar_url: expect.any(String),
+    };
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        console.log(users)
+        users.forEach((user) => {
```

Comment:
It's crucial to have a length check here before the beforeEach, otherwise if we were getting an empty array as a response the forEach would never run and therefore neither would the expect statements inside it giving us an auto pass on the test.